aws/resource_aws_wafv2_rule_group_test.go

```diff
@@ -937,7 +937,7 @@ func TestAccAwsWafv2RuleGroup_XssMatchStatement(t *testing.T) {
 	})
 }
 
-func testAccMatchResourceAttrArnWithIndexesAddr(name, format string, idx *int, arnService string, arnResourceRegexp *regexp.Regexp) resource.TestCheckFunc {
```

Comment:
This is okay for now, but I think we'll be looking to update this function for #13556 once that lands 👍